ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -53,7 +53,11 @@ export class ActivityNumericAnswer implements OnInit, OnChanges {
             this.numericField.setValue('');
             this.emitAnswer(null);
         } else if (this.numericField.invalid) {
-            this.emitAnswer(null);
```

Comment:
I've tracked this line down to [here](https://github.com/broadinstitute/ddp/pull/1103#issuecomment-545430633). It appears that there might have been a reason why we're emitting `null` here, but I'm not too sure if it still applies anymore.